README.md

```diff
@@ -1,6 +1,10 @@
 Commissaire
 ===========
-The lightweight REST API for Node Management.
+Commissaire is a lightweight REST interface for maintaining network hosts in a cluster through Ansible.
+
```

Comment:
"maintaining and managing"

I'd also add about being a central api point for other services.